src/gui/mzroll/mainwindow.cpp

```diff
@@ -1203,6 +1203,7 @@ TableDockWidget* MainWindow::addPeaksTable(QString title) {
         customTableId = std::atoi(idString.c_str());
     }
     TableDockWidget* panel = new PeakTableDockWidget(this, customTableId);
+	analytics->hitEvent("New Table", "Peak Table");
```

Comment:
There's another method used to add peak tables going by the name `MainWindow::createPeakTable`. Not sure if we should put GA on that one because its only used when loading mzroll tables saved from previous sessions. Just wanted to point this out.